src/CMakeLists.txt

```
@@ -96,7 +96,7 @@ if (${kokkos_DIR_FOUND})
   endif()
   target_compile_definitions(
     ${CHECKPOINT_LIBRARY} PUBLIC
-    KOKKOS_ENABLED_CHECKPOINT=1 KOKKOS_KERNELS_ENABLED=${KERNELS}
+    MAGISTRATE_ENABLED_KOKKOS=1 MAGISTRATE_ENABLED_KOKKOS_KERNELS=${KERNELS}
```

We usually go with `_ENABLED` at the end in `vt` (for example `vt_libfort_enabled`). Consider this for consistency:
```suggestion
    MAGISTRATE_KOKKOS_ENABLED=1 MAGISTRATE_KOKKOS_KERNELS_ENABLED=${KERNELS}
```

---

Fixed.